src/main/java/com/revature/services/EvalLogicImpl.java

```diff
@@ -16,40 +16,76 @@
 
 	@Autowired
 	private EvalRepository dao;
+	
+
+//CREATE----------------------------
 
+//RETRIEVE--------------------------
 	@Override
 	public Eval getEvalById(int id) {
 		Eval eval = dao.findOne(id);
-		System.out.println(eval);
+		try {
+			System.out.println(eval);
+		} catch (Exception e) {e.printStackTrace();}
 		return eval;
 	}
 	
 	@Override
-	public ArrayList<Eval> getEvalsByBatch(int id){
-		
-		ArrayList<Eval> evals = dao.findAllByBatchId(id);
+	public ArrayList<Eval> getEvalsByBatch(int id, String evalParam){
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1319)